dev/com.ibm.ws.microprofile.metrics/bnd.bnd

```diff
@@ -22,8 +22,6 @@ Bundle-Description: MicroProfile Metrics 1.0, version ${bVersion}
 IBM-Authorization-Roles: com.ibm.ws.management
```

Comment:
I'm not sure if this line and the one after should stay or not. Thoughts?

Reply:
I think @fmhwong  had worked this out before, so can hopefully comment on this

Reply:
`com.ibm.ws.management` has special meaning.  You cannot turn off authentication with special-subject EVERYONE.  If you change it to something else, it would break existing users as they need to explicit map the user to an administrator role even you use quickstartsecurity.

Reply:
Thanks for the info